gui_app.py

```diff
@@ -12,8 +12,7 @@
 
 from tkinter import Tk
 from tkinter import Toplevel
-from tkinter.filedialog import askdirectory
-from tkinter.filedialog import askopenfilename
+from tkinter import filedialog as fd
```

Comment:
L528's `askdirectory` function is orphaned with this change. I guess it just needs to be changed to `fd.askdirectory`.